project-management/Sprint_Reports/Sprint_1.md

```diff
@@ -0,0 +1,82 @@
+# Overview of CSP 
+
+## Functional Product:
+
+Adaptive Wireframes 
+https://c6q95y.axshare.com
+
+Experience Screens 
+https://xsswhw.axshare.com 
+
+UX Deliverables 
+https://blm1j3.axshare.com
+
+Tom's Review
+https://2huje4.axshare.com/
+
+## Board Issues
+
+### Issues Closed
+* #: Waiting on feedback from community on core MVP - Components outlined below
+*	#: 
+
+### Issues Progressed
+Progress was made on these tasks but they were not closed during the sprint.
+*	#: Cross Platform 
+*	#: Caregiver Platform Access - User Type / UX Deliverables Persona / User  Journey Explored 
+*	#: Interface Guidance 
```

Comment:
The # are supposed to be links to issues.... 

    [`#i`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/`i`): `issue title`

Reply:
they will be. I had some confusion, this will be updated as soon as I can.